submissions/madmaxWMFU/google-popup/index.html

```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>G00gle-style Popup</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="css/style.css">
+</head>
+
+<body>
+    <div class="container">
+        <div class="simpleNav">
+            <div class="divLinks">
+                <div><a class="link mail" href="#">Gmail</a></div>
+                <div><a class="link images" href="#">Images</a></div>
+            </div>
+            <div class="divApps">
+                <div>
+                    <label class="checker" for="applications" title="Applications"></label>
+                    <input id="applications" class="applications" type="checkbox" name="">
+                    <input id="moreApps" type="checkbox" name="">
+                    <div class="appsBox">
+                        <div class="mainApps">
+                            <div class="app">
+                                <a href="#">
+                                    <span id="myAccount" class="iconApp"></span>
+                                    <span class="textApp">My account</span>
+                                </a>
+                            </div>
+                            <div class="app">
+                                <a href="#">
+                                    <span class="iconApp" style="background-position:0 -1311px"></span>
```

Comment:
Keep in mind that inline styling is a bad approach for several reasons:

1. First of all, a browser will apply such styling for each line separately/ Which means, that every such line of code will be a reason for running of calculations of a page and for drawing it,  so you can receive a performance problem. Read about reflow and repaint.
2. This is an imperative way, you need to write declarative and describe what your code does, not how. This will make your code shorter and easier to maintain.
3. Reuse of code. Saying, you will need to rotate some other stuff - you will add a similar line to another part of an application. Which is not right because of DRY.
4. JS for logic, CSS - for styling.

Reply:
You can keep inline styling here as it is, just FYI.